src/components/link-detail/index.js

```diff
@@ -199,9 +200,13 @@ export class LinkDetail extends React.Component {
           >{link.lastWebhookSync && link.lastWebhookSync.status === 'TRIGGERED' ? 'Waiting...' : 'Resync'}</Button>
         </div>
 
-        {link.enabled && link.lastSyncedAt ? <div className="link-detail-row link-detail-last-sync-time">
-          <span>Last synced: <TimeAgo date={ link.lastSyncedAt } /></span>
-        </div> : null}
+        {link.enabled && link.lastSyncedAt ?
+          <div className="link-detail-row link-detail-last-sync-time">
+            <span>Last synced: <TimeAgo date={ link.lastSyncedAt } />
+              <span className="refresh-button" onClick={() => {this.props.onRefreshSync(link.id)}}>&#10227;</span>
```

Comment:
On my system, the refresh icon renders like this: <img width="39" alt="screen shot 2017-10-13 at 6 48 39 am" src="https://user-images.githubusercontent.com/1704236/31543078-9133ae7a-afe2-11e7-9c92-e36c5d8fe3d8.png">

At least at first, it wasn't clear to me that this was a refresh button. If you'd like, I can either make/find a svg icon you could use, or you can find something on your own. Let me know if you'd like me to put something together.

Also, would you be able to change the class name of the refresh button to be `link-detail-refresh-button`? I've been trying to namespace classes within the component that they are part of to try and prevent style cascading between components.